data/wp/wp-content/plugins/epfl/shortcodes/epfl-people/epfl-people.php

```diff
@@ -45,6 +45,13 @@ function epfl_people_2018_process_shortcode( $attributes, $content = null )
 
   ("" !== $units) ? $parameter['units'] = $units : $parameter['scipers'] = $scipers;
 
+  if (function_exists('pll_current_language')) {
+    $current_language = pll_current_language();
+    if(isset($current_language)) {
```

Comment:
Euh, je comprends pas la modif que t'as voulu faire là...
Pourquoi faire un `if(isset($current_language))` alors que tu viens d'initialiser la variable? dans tous les cas, ce test retournera `true`... elle ne pourrait retourner `false` que si la fonction `pll_current_language` retourne NULL, ce qui n'est pas possible selon la documentation de Polylang (https://polylang.wordpress.com/documentation/documentation-for-developers/functions-reference/)

De plus, là, si Polylang n'est pas installé, tu n'as pas de fallback. Et en plus, si c'est une langue autre que "en" ou "fr" qui est renvoyée, tu vas la récupérer et l'envoyer à People ?

Si jamais, l'exemple de code que j'avais fourni dans mon commentaire précédent, il fait tout ça et il est fonctionnel 😉

Reply:
uh, je découvre encore le php à mon age, c'est vrai que le isset est peut-être de trop, surtout à la lecture de la doc.

Pour le reste, j'envoie tel quel à people quoique ce soit, rien ou n'importe quoi, et je laisse people gérer selon la situation. C'est bulletproof pour le long terme comme cela.

Je simplifie